onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1545,6 +1545,11 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.args) > 0,
         reason="this ATen overload only supports one tensor as input by design",
     ),
+    TorchLibOpInfo(
+        "multinomial",
+        core_ops.aten_multinomial,
+        nondeterministic=True,
```

Comment:
```suggestion
        nondeterministic=True
```

so the lines can be formatted compactly